react-native/src/status_im/react_native/resources.cljs

```diff
@@ -15,3 +15,6 @@
    :ethlance           (js/require "./resources/images/contacts/ethlance.png")
    :commiteth          (js/require "./resources/images/contacts/commiteth.png")
    :etherplay          (js/require "./resources/images/contacts/etherplay.png")})
+
+(def assets
```

Comment:
imo  if we can't use svg icons we should use native images, because on iphones this image will be looking bad

Reply:
but maybe it's ok for now